src/amo/components/SearchForm/index.js

```diff
@@ -55,6 +57,7 @@ export class SearchFormBase extends React.Component {
 
     this.state = {
       searchValue: props.query || '',
+      lastAutocompleteFilters: {},
```

Comment:
Could this be either `previousAutoCompleteFilters` or `oldFilters`? Last sounds like the latest and is confusing, so I think this could be better named.

Reply:
Would `oldAutocompleteFilters` work for you? I started with `lastDispatched(Autocomplete)Filters`, but that was waaaay too long, so I shortened the name, but agreed that now it feels weird to have `last`.

Reply:
`oldAutocompleteFilters` works for me; we use `oldFilters` elsewhere. I think `previous` is better still but you're right it's verbose and we're already using the `old` prefix elsewhere so let's do that.